app/metrics.yaml

```diff
@@ -1776,3 +1776,70 @@ app_theme:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
+
+installation:
+  campaign:
+   type: string
```

Comment:
In order for me to help checking if this is the right type, can you provide an example for all these metrics?

Reply:
I did not find a way to trigger Adjust to populate these fields, I just used mock data to test them. So I cannot provide an example.

Reply:
Did someone provide you with mock data from Adjust?

Reply:
I'm afraid not.

Reply:
All right, @boek , do you know what is the shape of the data we're trying to collect here?